### PR TITLE
make course override values available in algolia

### PR DESCRIPTION
**Pull request by nickpizzo**
Jira issue: https://openedx.atlassian.net/browse/WS-2646

adds two new fields to algolia result fields, `organization_short_code_override` and `organization_logo_override`

### CODE REVIEW

File: course_discovery/apps/course_metadata/algolia_models.py [hunk 1]
```diff
@@ -216,6 +217,14 @@ def product_min_effort(self):
     def product_max_effort(self):
         return getattr(self.advertised_course_run, 'max_effort', None)
 
+    @property
+    def product_organization_short_code_override(self):
```
Comment (srwang) [inline course_discovery/apps/course_metadata/algolia_models.py:221]: What happens to programs? Does this field return null?

Have you tested indexing locally (to stage algolia) and checking that nothing breaks?

Comment (nickpizzo) [reply course_discovery/apps/course_metadata/algolia_models.py:221]: for visibility: paired with Sharon on locally testing the effect of all code changes on the algolia indexing process, all tests appeared to be successful

**Comment by DawoudSheraz** [comment]
@srwang Is this PR needed or should I close it out?

**Comment by DawoudSheraz** [comment]
Closing this, feel free to re-open or create a new one if this is still needed.

**Pull request closed without merging**